src/pages/en/tutorial/1-setup/1.mdx

```diff
@@ -26,15 +26,22 @@ You will use a **command line (terminal)** to create your Astro project and to r
 
 You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
 
-For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed. To check to see whether you already have a compatible version (`v16.12.0` or higher) installed: 
+### Node.js
 
-1. Launch your terminal and type the following into the command line:
+For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed, version `v16.12.0` or later.
 
-    ```sh
-    node -v
-    ```
+To check to see whether you already have a compatible version (`v16.12.0` or higher) installed, run the following command in your terminal:
```

Comment:
Thanks for this! One nit and we should be good to go!
Now that we provide the version in the first sentence, we don't need to repeat it: 

```suggestion
To check to see whether you already have a compatible version installed, run the following command in your terminal:
```